generic_config_updater/gu_common.py

```diff
@@ -212,6 +214,41 @@ def _invoke_validating_function(cmd, jsonpatch_element):
                 if not _invoke_validating_function(function, element):
                     raise IllegalPatchOperationError("Modification of {} table is illegal- validating function {} returned False".format(table, function))
 
+    def illegal_dataacl_check(self, old_config, upd_config):
```

Comment:
Suggest make the function name more descriptive. Illegal Data ACL is a very broad term. Maybe consider something like `check_acl_table_type_changed`. Going forward if there are more illegal changes we can create a check_illegal function and have it call this, which make it more structural and readable.

Reply:
Will update acltable to acl_table as suggested.
Here is the only case found in data acl. So I use this general definition. But if there are more, I will restructure the code here.